modules/local/concat_vcf.nf

```diff
@@ -15,8 +12,12 @@ process CONCAT_VCF {
     publishDir params.outdir, mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
 
-    conda environment
-    container container
+    conda (params.enable_conda ? "bioconda::htslib=1.11" : null)
```

Comment:
Does this also have a build number? ` --hd3b49d5_0` I mean?

Reply:
Good point, I'll look into it